app/constants/colors.js

```diff
@@ -11,6 +12,7 @@ const colors = {
   VIOLET_ALPHA_06: '#f7f8ff', // cannot use transparency
   VIOLET: '#4051DB',
   TRANSPARENT: 'transparent',
+  DARK_ALPHA: 'rgba(0, 0, 0, 0.7)',
```

Comment:
If we need alpha then its ok to use alpha. but ideally it should be a color defined by UX team and encoded into the `themes.js` so that there is a theme equivalent for the 3 themes.